packages/utilities/psammead-assets/CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Contributing Guide
+
+Please see our primary contributing guide which can be found at [the root of the Psammead respository](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md).
```

Comment:
Do we really want a Contributing file for every component??

Reply:
Happy to remove the file, what do you think @sareh?